File: EmployeeMangement.Data/Models/EmployeeModel.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace EmployeeMangement.Data.Models
{
    public partial class EmployeeModel
    {
      
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Gender { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string DateCreated { get; set; }
        public string DateModified { get; set; }
       
    }
}

File: EmployeeManagement-Web/ClientApp/src/app/admin/models/employee.model.ts
export class EmployeeAddModel {
    id?: number ; 
    firstName: string = "";
    lastName: string = "";
    gender: string = "";
    email: string = "";
    phone: string = "";
    dateCreated?: string = "";
    dateModified?: string = "";
    companyId: number = 0;
}